src/index.js

```diff
@@ -1,4 +1,9 @@
-import { factory, defaultConfig } from './Block';
-import { getDataProps } from './Block/data';
+import { createFactory } from './createFactory';
+import { factory as StarterBlock } from './block';
+import { styles } from './styles';
+import { configSchema, defaultConfig } from './blockConfig';
+import { getDataProps } from './data';
```

Comment:
Not a fan of having everything in the root directory. Let's talk about this 1x1?